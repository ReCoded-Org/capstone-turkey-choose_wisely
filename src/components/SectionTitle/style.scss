@import "../../style/variables.scss";
@import "../../style/mixins.scss";
.section_title {
  .big_title {
    color: #386641;
    font-weight: 400;
    font-size: 3.375rem;
    line-height: 3.955rem;
    letter-spacing: -0.031rem;
  }
  .subtitle {
    font-weight: normal;
    font-size: 1.125rem;
    line-height: 1.318rem;
    letter-spacing: -0.031rem;
    color: $gray-base-color;
    margin-bottom: 3.6rem;
    width: 100%;
    @include for-desktop-up {
      width: 70%;
    }
  }
  .title {
    color: #386641;
    font-weight: 300;
    font-size: 2.125rem;
    line-height: 2.875rem;
    letter-spacing: -0.00625rem;
  }
  .sub_text {
    font-weight: normal;
    font-size: 1.125rem;
    line-height: 1.625rem;
    letter-spacing: -0.03125rem;
    color: $secondary-color;
    margin-bottom: 3.6rem;
    width: 100%;
    @include for-desktop-up {
      width: 70%;
    }
  }
}
