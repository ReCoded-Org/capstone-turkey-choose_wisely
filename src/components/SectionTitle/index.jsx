import React from "react";
import "./style.scss";

const SectionTitle = ({ bigTitle, subtitle, title, subText }) => {
  return (
    <div className="section_title">
      <h1 className="big_title">{bigTitle && bigTitle}</h1>
      {subtitle && <p className="subtitle">{subtitle}</p>}
      <h2 className="title">{title && title}</h2>
      {subText && <p className="sub_text">{subText}</p>}
    </div>
  );
};

export default SectionTitle;
