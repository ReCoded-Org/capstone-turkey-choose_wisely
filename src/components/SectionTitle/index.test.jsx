import React from "react";
import SectionTitle from "./index";
import renderer from "react-test-renderer";

test("Snapshop test of sectionTitle component", () => {
  const map = renderer.create(<SectionTitle />);
  let tree = map.toJSON();

  expect(tree).toMatchSnapshot();
});
