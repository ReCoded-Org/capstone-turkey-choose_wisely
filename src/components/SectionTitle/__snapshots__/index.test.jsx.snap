// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop test of sectionTitle component 1`] = `
<div
  className="section_title"
>
  <h1
    className="big_title"
  />
  <h2
    className="title"
  />
</div>
`;
